### DISCUSSION TITLE
atom<Value | null>(null) is Atom, not PrimitiveAtom when `strict` is false in `tsconfig.json`

### DISCUSSION THREAD

**Discussion by himself65** [pmndrs/jotai#2259] [Bug report] [answered]
https://codesandbox.io/s/react-typescript-forked-ylp7kv

strict = true

<img width="663" alt="image" src="https://github.com/pmndrs/jotai/assets/14026360/184d496f-1952-47a8-b55f-db770f0d495f">

strict = false

<img width="688" alt="image" src="https://github.com/pmndrs/jotai/assets/14026360/eb607866-b96e-4b5f-8b5b-350dcaa2c819">

**Comment by dai-shi** [comment, marked as answer]
https://github.com/pmndrs/jotai/blob/234dff9c89642c6a1a724d2a73398119e4ea00fc/docs/guides/typescript.mdx#L12-L15
